src/main/java/com/twilio/oai/common/Utility.java

```diff
@@ -145,12 +145,4 @@ public Optional<CodegenModel> getModel(final List<CodegenModel> models,
     public Optional<CodegenModel> getModelByClassname(final List<CodegenModel> models, final String classname) {
         return models.stream().filter(model -> model.classname.equals(classname)).findFirst();
     }
-
-    public void addModel(final List<CodegenModel> allModels, final Map<String, CodegenModel> models, final String complexType, final String dataType) {
```

Comment:
I assumed we are removing it cause we are not using this function?

Reply:
Yeah, IDE showed no usage and it didn't break anything.